package.json

```diff
@@ -74,6 +74,7 @@
     "esbuild": "0.25.2",
     "graphql": "16.10.0",
     "graphql-rate-limit@npm:^3.3.0": "patch:graphql-rate-limit@npm%3A3.3.0#~/.yarn/patches/graphql-rate-limit-npm-3.3.0-3ae882ba75.patch",
+    "graphql-yoga": "5.13.3-alpha-20250407141542-4fbeb32b90471f85ba52de5d32894a7c6d79954b",
```

Comment:
let's make sure to use stable before merging

Reply:
Bumped to the latest stable 👍